R/integer64.R

```diff
@@ -2370,7 +2370,11 @@ seq.integer64 <- function(from=NULL, to=NULL, by=NULL, length.out=NULL, along.wi
     }
 
     if (!length.out){
-      length.out <- (to-from) %/% by + 1L
+      if (to == from) {
```

Comment:
IINM the crux here is that `by==0`, not that `to==from`, it's just that we've inferred `by==0` due to `to==from`.

Might it not be clearer for that to be the condition, then?
```suggestion
      if (by != 0L) {
```